cartography/intel/aws/lambda_function.py

```diff
@@ -29,47 +33,193 @@ def get_lambda_data(boto3_session: boto3.session.Session, region: str) -> List[D
 
 @timeit
 def load_lambda_functions(
-    neo4j_session: neo4j.Session, data: List[Dict], region: str, current_aws_account_id: str,
-    aws_update_tag: int,
+        neo4j_session: neo4j.Session, data: List[Dict], region: str, current_aws_account_id: str, aws_update_tag: int,
 ) -> None:
     ingest_lambda_functions = """
-    MERGE (lambda:AWSLambda{id: {Arn}})
+    UNWIND {lambda_functions_list} AS lf
+    MERGE (lambda:AWSLambda{id: lf.FunctionArn})
     ON CREATE SET lambda.firstseen = timestamp()
-    SET lambda.name = {LambdaName},
-    lambda.modifieddate = {LastModified},
-    lambda.arn = {Arn},
-    lambda.runtime = {Runtime},
-    lambda.description = {Description},
-    lambda.timeout = {Timeout},
-    lambda.memory = {MemorySize},
+    SET lambda.name = lf.FunctionName,
+    lambda.modifieddate = lf.LastModified,
+    lambda.runtime = lf.Runtime,
+    lambda.description = lf.Description,
+    lambda.timeout = lf.Timeout,
+    lambda.memory = lf.MemorySize,
+    lambda.codesize = lf.CodeSize,
+    lambda.handler = lf.Handler,
+    lambda.version = lf.Version,
+    lambda.tracingconfigmode = lf.TracingConfig.Mode,
+    lambda.revisionid = lf.RevisionId,
+    lambda.state = lf.State,
+    lambda.statereason = lf.StateReason,
+    lambda.statereasoncode = lf.StateReasonCode,
+    lambda.lastupdatestatus = lf.LastUpdateStatus,
+    lambda.lastupdatestatusreason = lf.LastUpdateStatusReason,
+    lambda.lastupdatestatusreasoncode = lf.LastUpdateStatusReasonCode,
+    lambda.packagetype = lf.PackageType,
+    lambda.signingprofileversionarn = lf.SigningProfileVersionArn,
+    lambda.signingjobarn = lf.SigningJobArn,
     lambda.lastupdated = {aws_update_tag}
-    WITH lambda
+    WITH lambda, lf
     MATCH (owner:AWSAccount{id: {AWS_ACCOUNT_ID}})
     MERGE (owner)-[r:RESOURCE]->(lambda)
     ON CREATE SET r.firstseen = timestamp()
     SET r.lastupdated = {aws_update_tag}
-    WITH lambda
-    MATCH (role:AWSPrincipal{arn: {Role}})
+    WITH lambda, lf
+    MATCH (role:AWSPrincipal{arn: lf.Role})
     MERGE (lambda)-[r:STS_ASSUME_ROLE_ALLOW]->(role)
     ON CREATE SET r.firstseen = timestamp()
     SET r.lastupdated = {aws_update_tag}
```

Comment:
Context after the UNWIND should be indented

Reply:
Ah I wasn't aware of having indentation after UNWIND. I'll make the change.